srcpkgs/haskell-language-server/template

```diff
@@ -0,0 +1,25 @@
+Template file for 'haskell-language-server'
+pkgname=haskell-language-server
+version=1.0.0
+revision=1
+# GHC only panics on i686
+# https://github.com/void-linux/void-packages/pull/27961/checks?check_run_id=2181519625#step:8:5936
+archs="~i686"
```

Comment:
Doesn't build for i686. I cannot send an issue to GitLab because I cannot sign up or sign in.

Please see the action for more information:
https://github.com/void-linux/void-packages/runs/2181519625?check_suite_focus=true#step:8:5936

Reply:
I can report if issue last after ghc update.